lib/taurus/qt/qtgui/taurusgui/taurusgui.py

```diff
@@ -58,6 +56,16 @@
 __docformat__ = 'restructuredtext'
 
 
+def QDoubleListDlg(*args, **kwargs):
```

Comment:
Same as in my previous comment, but in this case the solution is not as simple.

Still, I would prefer adding import lines where ExternalAppEditor or QDoubleListDlg are used.

Alternatively I would rename these functions to add a '_' prefix (and of course rename their usages accordingly).

Also, in this last case, I think that the functions should be clearly documented as "workarounds", and a TODO tag added to try to fix this by a refactoring of the imports.